frontend/src/lib/i18n/locales/en.json

```diff
@@ -207,6 +207,7 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "title": "Project filters",
       "show_deleted": "Show deleted",
       "hide_drafts": "Hide drafts",
+      "show_empty": "Show empty projects",
```

Comment:
We'd probably prefer "Show only empty projects" here. "Show empty projects" implies that empty projects are hidden by default and this filter option turns them visible, whereas what the filter actually does is hide all projects that are NOT empty. "Hide non-empty projects" would be another possibility, but that's a double negative, which tends to be confusing, so "Show only empty projects" is better language.

Reply:
@myieye Could we use your thought on this please? :)

Reply:
"Show only empty projects" is a decent candidate.
I think my first pick would be "Empty only".